.circleci/build_prod_config.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+import json
+
+with open('../dist/js-hash.json', 'r') as js_hash_file, open('../dist/css-hash.json', 'r') as css_hash_file:
+    js_hash = json.load(js_hash_file)
+    css_hash = json.load(css_hash_file)
+
+    # Bah, mutations
+    full_hash = js_hash.copy()
```

Comment:
You may want to use `deepcopy`, depending on what these hashes look like. `copy()` will only do the surface level – so objects passed by reference (so to speak) don’t get copied.

An example:

```pycon
>>> x = {'A': {'1': 1, '2': 2}}
>>> y = x.copy()
>>> x
{'A': {'1': 1, '2': 2}}
>>> y['A']['3'] = 3
>>> x
{'A': {'1': 1, '3': 3, '2': 2}}  # ???
```

Using `.deepcopy()` ensures you really do get distinct copies of everything.

Reply:
This should be a 1 dimensional array (`{fileKey => hashedFile}`), I'd like to keep it that way... good to know though.